### DISCUSSION TITLE
Public qualifier for tables in view definition lost when schema.rb is updated

### DISCUSSION THREAD

**Discussion by Revolution1157** [scenic-views/scenic#418] [Q&A]
I have been using the scenic-view gem for a number of years. I just encounter an issue that may present a problem for others.  Our application is a multi-tenant based application using a unique schema for each client. The issue occurs when i add a new client. Tables and views are created based on the schema.rb file but unfortunately any view that includes a "public".table in its definition is generated incorrectly. The schema.rb file does not include the "public" qualification. Am I overlooking something when i create new views or is this something that needs to be dealt with each time i create a new client / postgres schema?

**Comment by dlupu** [comment]
@Revolution1157 we are running into exactly the same issue. I was wondering it you found a solution for this problem.
